site/en/r2/tutorials/eager/custom_training.ipynb

```diff
@@ -246,22 +263,22 @@
       "source": [
         "### Define a loss function\n",
         "\n",
-        "A loss function measures how well the output of a model for a given input matches the desired output. Let's use the standard L2 loss."
+        "A loss function measures how well the output of a model for a given input matches the target output. It is like an objective function, the quantity of which will be minimized during training. Let's use the standard L2 loss, also known as the least square errors:"
```

Comment:
@MarkDaoust *objective function* isn't a term I've seen in the TF docs and not sure it clarifies here. wdyt?

Reply:
@MarkDaoust @lamberta from the theory I understand that loss function = error function = objective function (as mentioned in DeepMind's Andrew Trask's excellent Grokking Deep Learning book). Perhaps, keeping it simple is better: `"The goal is to minimize it during training"`. This is still useful for those folks who are new to ML imo.

Reply:
Yeah, I like that idea. Let's remove the object function term and keep your explanation. Thanks